### PR TITLE
Depoyment

### PR DESCRIPTION
**Pull request by Jan11979**
Deployment mit MongoBD

### CODE REVIEW

File: backend/src/main/java/com/example/backandfront/controller/BasicController.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.math.BigInteger;
+import java.text.SimpleDateFormat;
+import java.util.AbstractList;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("api")
+@CrossOrigin(origins = "*", allowedHeaders = "*")
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/example/backandfront/controller/BasicController.java:18]: Is this nessecary?

File: backend/src/main/java/com/example/backandfront/controller/ShoppingDBRepository.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.example.backandfront.controller;
+
+import com.example.backandfront.model.ShoppingItem;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+import java.math.BigInteger;
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/example/backandfront/controller/ShoppingDBRepository.java:6]:
```suggestion

```

File: backend/src/main/java/com/example/backandfront/model/ShoppingItem.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.example.backandfront.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+import java.math.BigInteger;
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/example/backandfront/model/ShoppingItem.java:8]:
```suggestion
```

File: backend/src/main/java/com/example/backandfront/model/ShoppingItem.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ShoppingItem {
+    @Id
+    String  id;
+    String  name;
```
Comment (Flooooooooooorian) [inline backend/src/main/java/com/example/backandfront/model/ShoppingItem.java:16]:
```suggestion
   private  String  id;
   private String  name;
```

File: frontend/src/Controller/Fetching.ts [hunk 5]
```diff
@@ -1,9 +1,11 @@
 import axios from "axios";
```
Comment (Flooooooooooorian) [inline frontend/src/Controller/Fetching.ts:2]: Maybe have a look at https://create-react-app.dev/docs/proxying-api-requests-in-development/

**Review by Flooooooooooorian — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/example/backandfront/controller/BasicController.java (hunk 1 above)
Comment (Jan11979) [reply backend/src/main/java/com/example/backandfront/controller/BasicController.java:18]: OK

Comment (Jan11979) [inline backend/src/main/java/com/example/backandfront/controller/BasicController.java:18]:
```suggestion

```

File: backend/src/main/java/com/example/backandfront/controller/BasicController.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        SimpleDateFormat formatter= new SimpleDateFormat("yyyy-MM-dd 'at' HH:mm:ss z");
+        Date date = new Date(System.currentTimeMillis());
+
+
+        List<String> tmpList = new ArrayList<>();
+        tmpList.add( "Miau um: " + formatter.format(date));
+        tmpList.add( "Service: BackAndFront " );
+//        tmpList.add( "spring.data.mongodb.database: " + mongdatabase );
+//        tmpList.add( "spring.data.mongodb.host: " + monghost );
+//        tmpList.add( "spring.data.mongodb.port: " + mongport );
```
[the remark this answers is not in the document]
Comment (Jan11979) [reply backend/src/main/java/com/example/backandfront/controller/BasicController.java:38]: I wan't to keep it 
Testing

**Pull request merged**